src/app/App.css

```diff
@@ -15,6 +15,7 @@
     & .page {
         width: calc(100% - 8rem);
         margin-left: 8rem;
+        display: flex;
```

Comment:
Why did we need this?

Reply:
We needed this because any child of the `.page` element cannot use `height: 100%`, to occupy the full viewport, given that `.page` has `flex: 1 0 auto` and thus no defined height